catalog/video/video-mpegts-transformer/src/main/java/org/codice/alliance/transformer/video/MpegTsInputTransformer.java

```diff
@@ -136,11 +146,26 @@ public Metacard transform(InputStream inputStream, final String id)
 
             extractStanag4609Metadata(metacard, fileBackedOutputStream);
 
+            extractMediaEncodings(metacard, fileBackedOutputStream);
+
             return metacard;
         }
 
     }
 
+    private void extractMediaEncodings(Metacard metacard, TemporaryFileBackedOutputStream fbos)
+            throws IOException {
+
+        List<Serializable> serializables = TSStream.from(fbos.asByteSource())
+                .map(PESPacket::getStreamType)
+                .distinct()
+                .map(streamTypeToString)
+                .collect(Collectors.<Serializable>toList());
+
+        metacard.setAttribute(new AttributeImpl(AttributeNameConstants.MEDIA_ENCODING,
```

Comment:
That may be, but it's being done in some other places as well : 
https://github.com/codice/ddf/blob/92090e0c33e6cca5970e02617099373158fb9e2b/catalog/spatial/registry/registry-ebrim-transformer/src/main/java/org/codice/ddf/registry/converter/RegistryPackageConverter.java#L329

https://github.com/codice/ddf/blob/master/catalog/spatial/csw/spatial-csw-transformer/src/main/java/org/codice/ddf/spatial/ogc/csw/catalog/transformer/GmdTransformer.java#L359